staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -54,6 +54,13 @@ const (
 	PatchAuditAnnotationPrefix = "patch.webhook.admission.k8s.io/"
 	// MutationAuditAnnotationPrefix is a prefix for presisting webhook mutation existence in audit annotation.
 	MutationAuditAnnotationPrefix = "mutation.webhook.admission.k8s.io/"
+	// FailureAuditAnnotationPrefix is a prefix for keeping noteworthy
+	// failure audit annotations.
+	FailureAuditAnnotationPrefix = "failure.webhook.admission.k8s.io/"
```

Comment:
the prefix would be indistinguishable if there are one validating webhook and one mutating webhook sharing the same name (but having overlapping rules), but that could be rare as well. I don't feel strongly about it

Reply:
Ack.  I prefer just to leave it as is, I think it's a pretty rare situation.